lib/sym/data/wrapper_struct.rb

```diff
@@ -9,6 +9,7 @@ class WrapperStruct < Struct.new(
       :version,               # [Integer] Version of the cipher used
       :compress               # [Boolean] indicates if compression should be applied
       )
+      define_singleton_method(:new, Class.method(:new))
```

Comment:
What’s this for?

Reply:
This is the hack that I referenced in the PR.  There is currently an issue with overriding the initializer to take different argument types.  This specific line was taken from the suggestion in https://github.com/ruby/ruby/pull/3045#issuecomment-625707122

I selected this hack since it was the smallest change.  Alternatively, the struct can be created with `keyword_init: true`.  I'm happy to provide that implementation if you'd prefer.

Reply:
No this is fine, thanks for the context. Let me review it and test the branch locally, then I’ll merge it.

Greatly appreciate your contribution.